.changeset/several-cucumbers-wonder.md

```diff
@@ -0,0 +1,5 @@
+Added: `declaration-property-value-disallowed-list` `messageArgs` support
+Added: `declaration-property-unit-disallowed-list` `messageArgs` support
+Added: `function-disallowed-list` `messageArgs` support
+Added: `at-rule-disallowed-list` `messageArgs` support
+Added: `property-disallowed-list` `messageArgs` support
```

Comment:
[suggest] Thanks for adding the entry. Could you please update it as follows to work `changeset` later?

```suggestion
---
"stylelint": minor
---

Added: custom message formatting for `at-rule-disallowed-list`, `declaration-property-unit-disallowed-list`, `declaration-property-value-disallowed-list`, `function-disallowed-list`, and `property-disallowed-list`
```